source/features/faster-pr-diff-options.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
-import {BookIcon, CheckIcon, DiffIcon} from '@primer/octicons-react';
+import {BookIcon, CheckIcon, DiffIcon, DiffModifiedIcon} from '@primer/octicons-react';
 
 import features from '.';
 
+function isPRPage(): boolean {
+	return !pageDetect.isSingleCommit() && !pageDetect.isCompare();
```

Comment:
Can you use PR-related checks instead of exclusions? These 2 alone don't ensure it's a PR page